src/IKVM.Runtime/TypeWrapper.cs

```diff
@@ -3605,6 +3605,11 @@ internal virtual byte[] GetFieldRawTypeAnnotations(FieldWrapper fw)
             return null;
         }
 
+        public bool Equals(TypeWrapper other)
```

Comment:
This can't be right. The name of a type wrapper isn't guaranteed to be unique, as the name of types themselves aren't guaranteed to be unique.

Reply:
How it should be organized? There is area that can contains only unique type names?

That code was written for comparing already loaded TypeWrapper and pending to load. Were encountered some problems with loading library classes from another library.

Reply:
No. There is no such thing as unique type names. Type names can be duplicated.